platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/linking/InOrderLinker.java

```diff
@@ -132,17 +136,34 @@ public InOrderLinker(
                         new LongAccumulator.Config(PLATFORM_CATEGORY, "parentGenerationMismatch")
                                 .withDescription(
                                         "Parent child relationships where claimed parent generation did not match actual parent generation"));
+        this.birthRoundMismatchAccumulator = platformContext
+                .getMetrics()
+                .getOrCreate(
+                        new LongAccumulator.Config(PLATFORM_CATEGORY, "parentBirthRoundMismatch")
+                                .withDescription(
+                                        "Parent child relationships where claimed parent birth round did not match actual parent birth round"));
         this.timeCreatedMismatchAccumulator = platformContext
                 .getMetrics()
                 .getOrCreate(
                         new LongAccumulator.Config(PLATFORM_CATEGORY, "timeCreatedMismatch")
                                 .withDescription(
                                         "Parent child relationships where child time created wasn't strictly after parent time created"));
 
+        final AncientMode ancientMode = platformContext
+                .getConfiguration()
+                .getConfigData(EventConfig.class)
+                .getAncientMode();
         this.nonAncientEventWindow = NonAncientEventWindow.getGenesisNonAncientEventWindow(platformContext
                 .getConfiguration()
                 .getConfigData(EventConfig.class)
                 .getAncientMode());
```

Comment:
`getAncientMode()` via config here can be replaced with the local `ancientMode` variable

Reply:
replaced.